content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst

```diff
@@ -33,117 +34,95 @@ SEPA Direct Debit Mandates
-process all payments at once.
+`.xml` files with all SDD payment instructions can be uploaded to your online banking interface
+to process all payments at once.
 
 .. note::
    The files generated by Odoo follow the SEPA Direct Debit **PAIN.008.001.02** specifications, as
-   required by the SEPA customer-to-bank Implementation Guidelines, which ensures compatibility with
-   the banks.
-
-To generate your XML file for the pending SDD payments, go to the related *bank journal* on your
-*Accounting dashboard*, then click on *Direct Debit Payments to Collect*.
-
-.. image:: batch_sdd/batch-sdd-collect.png
-   :align: center
-   :alt: Start the SDD payments collection in Odoo Accounting
-
-Select all the payments you want to include in your SDD XML file, then click on *Action* and select
-*Create Batch Payment*.
-
-.. image:: batch_sdd/batch-sdd-batch.png
-   :align: center
-   :alt: Create a batch payment with all the SDD payments in Odoo Accounting
+   required by the SEPA customer-to-bank `implementation guidelines
+   <https://www.europeanpaymentscouncil.eu/document-library/implementation-guidelines/sepa-credit-transfer-customer-psp-implementation>`_,
+   which ensures compatibility with the banks.
 
-Odoo then takes you to your *Batch Payment*’s form. Click on *Validate* and download the SDD XML
-file.
+To generate your `.xml` file of multiple pending SDD payments, you can create a batch payment.
+To do so, go to :menuselection:`Accounting app --> Customers --> Payments`, select the needed
+payments, then click on :guilabel:`Action`, and finally :guilabel:`Create a batch Payment`. Once
+you click on :guilabel:`Validate`, the `.xml` file is directly available for download.
 
-.. image:: batch_sdd/batch-sdd-xml.png
+.. image:: batch_sdd/xml.png
    :align: center
-   :alt: Generate an XML file for your SDD payments in Odoo Accounting
+   :alt: Generate an .XML file for your SDD payments in Odoo Accounting
 
 Finally, upload this file to your online banking interface to process the payments.
 
 .. tip::
-   You can retrieve all the generated SDD XML files by going to
-   :menuselection:`Accounting --> Configuration --> Customers --> Batch Payments`.
+   You can retrieve all the generated SDD `.xml` files by going to
+   :menuselection:`Accounting app --> Customers --> Batch Payments`.
 
 .. seealso::
    * :doc:`batch`
    * :doc:`../../bank/setup/bank_accounts`
-   * `Odoo Academy: SEPA Direct Debit Mandates (SDD) <https://www.odoo.com/r/Zxs>`_
    * `List of all SEPA countries
-     <https://www.europeanpaymentscouncil.eu/document-library/other/epc-list-sepa-scheme-countries>`_
\ No newline at end of file
+     <https://www.europeanpaymentscouncil.eu/document-library/other/epc-list-sepa-scheme-countries>`_
+   * `Sepa guidelines
+     <https://www.europeanpaymentscouncil.eu/document-library/implementation-guidelines/sepa-credit-transfer-inter-psp-implementation-guidelines>`_
```

Comment:
This should probably be written somewhere, but the guideline is to always have exactly 1 blank line at the end of each file. That's true for many languages (not only RST). This helps editors highlight the code correctly.
```suggestion
     <https://www.europeanpaymentscouncil.eu/document-library/implementation-guidelines/sepa-credit-transfer-inter-psp-implementation-guidelines>`_

```